firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -400,6 +400,41 @@ private boolean isAuthTokenExpired(PersistedFidEntry persistedFidEntry) {
   private long currentTimeInSecs() {
     return TimeUnit.MILLISECONDS.toSeconds(clock.currentTimeMillis());
   }
+
+  /**
+   * Deletes the firebase installation id of the {@link FirebaseApp} from FIS servers and local
+   * storage.
+   */
+  private Void deleteFirebaseInstallationId() throws FirebaseInstallationsException {
+
+    PersistedFidEntry persistedFidEntry = persistedFid.readPersistedFidEntryValue();
+
+    if (!isPersistedFidRegistered(persistedFidEntry)) {
+      persistedFid.clear();
+      return null;
+    }
+
+    deleteFID(persistedFidEntry);
```

Comment:
Wondering if it would be clearer to inline the deleteFID method here:

```
    if (isPersistedFidRegistered(persistedFidEntry)) {
          try {
            serviceClient.deleteFirebaseInstallation(/* args */);
          } catch (FirebaseInstallationServiceException exception) {
            throw new FirebaseInstallationsException(
              "Failed to delete a Firebase Installation.",
              FirebaseInstallationsException.Status.SDK_INTERNAL_ERROR);
          }
    }
    persistedFid.clear();
    return null;
```

Reply:
Done. Thanks.